tox.ini

```diff
@@ -32,14 +32,26 @@ deps = https://github.com/willkg/dennis/archive/aca57f6d.zip
 commands = dennis-cmd lint --errorsonly locale/
 
 [testenv:docker]
-commands = 
-    docker-compose -f docker-compose.yml -f scripts/docker-compose.travis.yml up -d
-    docker exec -i kuma_web_1 make compilejsi18n collectstatic
+whitelist_externals =
+    docker
+    docker-compose
+    make
+    sleep
+setenv =
+    COMPOSE_FILE = docker-compose.yml:scripts/docker-compose.travis.yml
+    COMPOSE_PROJECT_NAME = kumatox
+commands =
+    make build-base VERSION=latest
+    docker-compose build mysql
```

Comment:
Yes, but spits out [a warning](https://travis-ci.org/mozilla/kuma/jobs/166060719#L390):

```
WARNING: Image for service mysql was built because it did not already exist. To rebuild this image you must use `docker-compose build` or `docker-compose up --build`.
```

My viewpoint is "when I look at a failed build log, what will jump out at me?" This warning will, and I'll have to ignore it, and train other devs to ignore it. Or, add a build step and eliminate it, so that only the build failure stands out.